tests/net/all/prj.conf

```diff
@@ -106,8 +106,6 @@ CONFIG_NET_CONTEXT_SYNC_RECV=y
 CONFIG_NET_CONTEXT_CHECK=y
 
 # SLIP
-CONFIG_NET_SLIP_TUN=n
-CONFIG_NET_SLIP_TAP=y
```

Comment:
I'm looking at introducing a CONFIG_COMPILE_TEST that we can use to handle such cases going forward.